DemoqaProject/tests/AlertsTest.cs

```diff
@@ -0,0 +1,18 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class AlertsTest : BaseSeleniumTest
+    {
+        [Test]
+        public void CheckAlertsText()
+        {
+            String getHeader = Config.alertsText;
+            HomePage homePage = new HomePage();
+            Alerts alerts = new Alerts();
```

Comment:
When you done with header method refactoring you won't need to create here alerts and other pages objects. Don't forget to remove it from after.

Reply:
It's actual for other pages as well.